src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
 
+        if (fromFileName == null) {
+            throw new IllegalArgumentException("File is not found!");
+        }
```

Comment:
The error message 'File is not found!' is misleading when checking for null. Consider changing it to something like 'Input file name is null!' to better reflect the actual issue.